orc8r/gateway/python/magma/common/redis/containers.py

```diff
@@ -313,14 +313,14 @@ def __delitem__(self, key: str) -> int:
             raise KeyError(composite_key)
         return deleted_count
 
-    def get(self, key: str) -> Optional[T]:
+    def get(self, key: str, default=None) -> Optional[T]:
         """Get ``d[key:type]`` from dictionary.
         Returns None if *key:type* is not in the map
         """
         try:
             return self.__getitem__(key)
         except (KeyError, ValueError):
-            return None
+            return default
```

Comment:
This is changing the contract of this function. As long as it's passing CI it should be fine, but considering AGW's low test coverage it may be worth double-checking usages and whether any of them are explicitly checking for KeyError etc

Reply:
How is it changing contract? This is adding optional parameter with same return value when arg is not specified.